### DISCUSSION TITLE
Wordpress

### DISCUSSION THREAD

**Discussion by jdmba** [fancyapps/ui#262] [Q&A]
I see that the prior discussion on getting v4.x to work with Wordpress wound up not being finished :(
https://github.com/fancyapps/ui/discussions/42

Would you please consider providing official instructions to get this working with Wordpress?

**Comment by fancyapps** [comment]
Hi,

Sorry, there are so many feature requests and so much work to do that it's hard to handle everything. 

Basically, installation instructions work on any site - https://fancyapps.com/docs/ui/installation#cdn - it does not matter if you are using Wordpress or any other CMS.

Then, if you can't add `data-fancybox` attributes to your links, you can use the generic selector to attach Fancybox, for example:

```js
Fancybox.bind('a[href*=".jpg"],a[href*=".jpeg"], a[href*=".png"],a[href*=".gif"]', {
  // Your options go here
  groupAll: true 
});
```

**Reply by jdmba** [reply]
Thank you for your reply.  It is not so easy as what you say, but I have figured it out.  Whether you want to test it and put it into your instructions I leave to you, but this is how I successfully got it to work in wordpress:

1) From the WORDPRESS DASHBOARD go to CUSTOMIZE and then ADDITIONAL CSS
2) Copy and paste the .css file into the window and click SAVE
3) Install SCRIPTS AND STYLES plug in
4) In the section "Scripts (end of the body tag):" copy and paste the .js file and click SAVE CHANGES

That's it.   This is substandard because to update you need to copy/paste, but it is the only way I have found to get this to work inside Wordpress
